#general
base_frame_id: base_link
odom_frame_id: odom
wheel_diameter: 0.0415
wheel_separation: 0.26
frequency: 20
#encoder
encoder_pulses_rev: 1400
#PID
kp: 2 #0.9
ki: 0.4 #0.3
kd: 0
integrator_max: 2
pid_max: 1      #max 1
pid_min: 0.2    #set min PWM signal that is required to make motor move to avoid 
